System/Process/Env/OsString.hs

```diff
@@ -0,0 +1,32 @@
+{-# LANGUAGE CPP #-}
+
+#if !MIN_VERSION_filepath(1,5,0)
+-- Silence the warning on System.OsString.Internal.Types import. It exists
+-- (deprecated) in filepath 1.4.100.0 && < 1.5.0.0. For filepath >= 1.5,
```

Comment:
```suggestion
-- (deprecated) in filepath >= 1.4.100.0 && < 1.5.0.0. For filepath >= 1.5,
```

Is there a missing `>=` here?

Reply:
This doesn't seem to have been resolved. Has a push gone astray?

Reply:
Possibly? I removed the entire comment per [your comment](https://github.com/haskell/process/pull/339#discussion_r2034063116). In any case, it's gone now.